
2024-06-13 21:27:46 [DEBUG]: Cleaning output
2024-06-13 21:27:46 [DEBUG]: jpn_vert
2024-06-13 21:27:46 [ERROR]: Unsupported language
2024-06-13 21:27:47 [DEBUG]: Cleaning output
2024-06-13 21:27:47 [DEBUG]: jpn_vert
2024-06-13 21:27:47 [ERROR]: Unsupported language
2024-06-13 21:27:47 [DEBUG]: Cleaning output
2024-06-13 21:27:47 [DEBUG]: jpn_vert
2024-06-13 21:27:47 [ERROR]: Unsupported language
2024-06-13 21:27:47 [DEBUG]: Cleaning output
2024-06-13 21:27:47 [DEBUG]: jpn_vert
2024-06-13 21:27:47 [ERROR]: Unsupported language
2024-06-13 21:27:48 [DEBUG]: Cleaning output
2024-06-13 21:27:48 [DEBUG]: jpn_vert
2024-06-13 21:27:48 [ERROR]: Unsupported language
2024-06-13 21:27:48 [DEBUG]: Cleaning output
2024-06-13 21:27:48 [DEBUG]: jpn_vert
2024-06-13 21:27:48 [ERROR]: Unsupported language
2024-06-13 21:27:49 [DEBUG]: Cleaning output
2024-06-13 21:27:49 [DEBUG]: jpn_vert
2024-06-13 21:27:49 [ERROR]: Unsupported language
2024-06-13 21:27:49 [DEBUG]: Cleaning output
2024-06-13 21:27:49 [DEBUG]: jpn_vert
2024-06-13 21:27:49 [ERROR]: Unsupported language
2024-06-13 21:27:49 [DEBUG]: Cleaning output
2024-06-13 21:27:49 [DEBUG]: jpn_vert
2024-06-13 21:27:49 [ERROR]: Unsupported language
2024-06-13 21:27:50 [DEBUG]: Cleaning output
2024-06-13 21:27:50 [DEBUG]: jpn_vert
2024-06-13 21:27:50 [ERROR]: Unsupported language
2024-06-13 21:27:50 [DEBUG]: Cleaning output
2024-06-13 21:27:50 [DEBUG]: jpn_vert
2024-06-13 21:27:50 [ERROR]: Unsupported language
2024-06-13 21:27:50 [DEBUG]: Cleaning output
2024-06-13 21:27:50 [DEBUG]: jpn_vert
2024-06-13 21:27:50 [ERROR]: Unsupported language
2024-06-13 21:27:51 [DEBUG]: Cleaning output
2024-06-13 21:27:51 [DEBUG]: jpn_vert
2024-06-13 21:27:51 [ERROR]: Unsupported language
2024-06-13 21:27:51 [DEBUG]: Cleaning output
2024-06-13 21:27:51 [DEBUG]: jpn_vert
2024-06-13 21:27:51 [ERROR]: Unsupported language
2024-06-13 21:27:52 [DEBUG]: Cleaning output
2024-06-13 21:27:52 [DEBUG]: jpn_vert
2024-06-13 21:27:52 [ERROR]: Unsupported language
2024-06-13 21:27:52 [DEBUG]: Cleaning output
2024-06-13 21:27:52 [DEBUG]: jpn_vert
2024-06-13 21:27:52 [ERROR]: Unsupported language
2024-06-13 21:27:52 [DEBUG]: Processing blurb number 0/39
2024-06-13 21:27:52 [DEBUG]: Text before translation : トーイ _
2024-06-13 21:27:52 [ERROR]: name 'groq_translate' is not defined
2024-06-13 21:27:52 [ERROR]: トーイ _
2024-06-13 21:27:52 [Level 100]: An exception occurred
Traceback (most recent call last):
  File "C:\Users\Skull\Documents\github\Open-Translation\trad.py", line 624, in <module>
    main()
  File "C:\Users\Skull\Documents\github\Open-Translation\trad.py", line 553, in main
    raw_translated_text = str(translated.translation.decode("utf-8", 'ignore'))
AttributeError: 'str' object has no attribute 'decode'
2024-06-13 21:27:53 [DEBUG]: Script starting
2024-06-13 21:27:53 [DEBUG]: Namespace(input_language='japanese', output_language='en', input_folder='C:\\Users\\Skull\\Documents\\github\\OpenManga/public/media/171/005.jpg', output_folder='C:\\Users\\Skull\\Documents\\github\\OpenManga/public/media/171/translated/', transparency=200, json=True)
2024-06-13 21:27:53 [INFO]: arguments : -if C:\Users\Skull\Documents\github\OpenManga/public/media/171/005.jpg -of C:\Users\Skull\Documents\github\OpenManga/public/media/171/translated/ -t 200 i jpn_vert o en json True
2024-06-13 21:27:53 [INFO]: input path is a file
2024-06-13 21:27:53 [DEBUG]: True
2024-06-13 21:27:53 [DEBUG]: full translation
2024-06-13 21:27:53 [DEBUG]: Cleaning output
2024-06-13 21:27:53 [DEBUG]: jpn_vert
2024-06-13 21:27:53 [ERROR]: Unsupported language
2024-06-13 21:27:54 [DEBUG]: Cleaning output
2024-06-13 21:27:54 [DEBUG]: jpn_vert
2024-06-13 21:27:54 [ERROR]: Unsupported language
2024-06-13 21:27:55 [DEBUG]: Cleaning output
2024-06-13 21:27:55 [DEBUG]: jpn_vert
2024-06-13 21:27:55 [ERROR]: Unsupported language
2024-06-13 21:27:56 [DEBUG]: Cleaning output
2024-06-13 21:27:56 [DEBUG]: jpn_vert
2024-06-13 21:27:56 [ERROR]: Unsupported language
2024-06-13 21:27:57 [DEBUG]: Cleaning output
2024-06-13 21:27:57 [DEBUG]: jpn_vert
2024-06-13 21:27:57 [ERROR]: Unsupported language
2024-06-13 21:27:58 [DEBUG]: Cleaning output
2024-06-13 21:27:58 [DEBUG]: jpn_vert
2024-06-13 21:27:58 [ERROR]: Unsupported language
2024-06-13 21:27:59 [DEBUG]: Cleaning output
2024-06-13 21:27:59 [DEBUG]: jpn_vert
2024-06-13 21:27:59 [ERROR]: Unsupported language
2024-06-13 21:27:59 [DEBUG]: Cleaning output
2024-06-13 21:27:59 [DEBUG]: jpn_vert
2024-06-13 21:27:59 [ERROR]: Unsupported language
2024-06-13 21:28:00 [DEBUG]: Cleaning output
2024-06-13 21:28:00 [DEBUG]: jpn_vert
2024-06-13 21:28:00 [ERROR]: Unsupported language
2024-06-13 21:28:00 [DEBUG]: Cleaning output
2024-06-13 21:28:00 [DEBUG]: jpn_vert
2024-06-13 21:28:00 [ERROR]: Unsupported language
2024-06-13 21:28:00 [DEBUG]: Cleaning output
2024-06-13 21:28:00 [DEBUG]: jpn_vert
2024-06-13 21:28:00 [ERROR]: Unsupported language
2024-06-13 21:28:01 [DEBUG]: Cleaning output
2024-06-13 21:28:01 [DEBUG]: jpn_vert
2024-06-13 21:28:01 [ERROR]: Unsupported language
2024-06-13 21:28:01 [DEBUG]: Cleaning output
2024-06-13 21:28:01 [DEBUG]: jpn_vert
2024-06-13 21:28:01 [ERROR]: Unsupported language
2024-06-13 21:28:02 [DEBUG]: Cleaning output
2024-06-13 21:28:02 [DEBUG]: jpn_vert
2024-06-13 21:28:02 [ERROR]: Unsupported language
2024-06-13 21:28:02 [DEBUG]: Cleaning output
2024-06-13 21:28:02 [DEBUG]: jpn_vert
2024-06-13 21:28:02 [ERROR]: Unsupported language
2024-06-13 21:28:02 [DEBUG]: Cleaning output
2024-06-13 21:28:02 [DEBUG]: jpn_vert
2024-06-13 21:28:02 [ERROR]: Unsupported language
2024-06-13 21:28:03 [Level 100]: An exception occurred
Traceback (most recent call last):
  File "C:\Users\Skull\Documents\github\Open-Translation\trad.py", line 624, in <module>
    main()
  File "C:\Users\Skull\Documents\github\Open-Translation\trad.py", line 544, in main
    blurbs, height, width = get_blurbs(img, input_language, ocr_mode)
  File "C:\Users\Skull\Documents\github\Open-Translation\trad.py", line 316, in get_blurbs
    text = pytesseract.image_to_string(pil_image, lang=input_language, config=get_params(ocr_mode))
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 423, in image_to_string
    return {
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 426, in <lambda>
    Output.STRING: lambda: run_and_get_output(*args),
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 288, in run_and_get_output
    run_tesseract(**kwargs)
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 262, in run_tesseract
    with timeout_manager(proc, timeout) as error_string:
  File "C:\Python310\lib\contextlib.py", line 135, in __enter__
    return next(self.gen)
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 127, in timeout_manager
    yield proc.communicate()[1]
  File "C:\Python310\lib\subprocess.py", line 1149, in communicate
    stdout, stderr = self._communicate(input, endtime, timeout)
  File "C:\Python310\lib\subprocess.py", line 1523, in _communicate
    self.stdout_thread.join(self._remaining_time(endtime))
  File "C:\Python310\lib\threading.py", line 1089, in join
    self._wait_for_tstate_lock()
  File "C:\Python310\lib\threading.py", line 1109, in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
KeyboardInterrupt
2024-06-13 21:58:02 [DEBUG]: Script starting
2024-06-13 21:58:02 [DEBUG]: Namespace(input_language='japanese', output_language='en', input_folder='C:\\Users\\Skull\\Documents\\github\\OpenManga/public/media/171/006.jpg', output_folder='C:\\Users\\Skull\\Documents\\github\\OpenManga/public/media/171/translated/', transparency=200, json=True)
2024-06-13 21:58:02 [INFO]: arguments : -if C:\Users\Skull\Documents\github\OpenManga/public/media/171/006.jpg -of C:\Users\Skull\Documents\github\OpenManga/public/media/171/translated/ -t 200 i jpn_vert o en json True
2024-06-13 21:58:02 [INFO]: input path is a file
2024-06-13 21:58:02 [DEBUG]: True
2024-06-13 21:58:02 [DEBUG]: full translation
2024-06-13 21:58:03 [DEBUG]: Cleaning output
2024-06-13 21:58:03 [DEBUG]: jpn_vert
2024-06-13 21:58:03 [ERROR]: Unsupported language
2024-06-13 21:58:03 [DEBUG]: Cleaning output
2024-06-13 21:58:03 [DEBUG]: jpn_vert
2024-06-13 21:58:03 [ERROR]: Unsupported language
2024-06-13 21:58:04 [DEBUG]: Cleaning output
2024-06-13 21:58:04 [DEBUG]: jpn_vert
2024-06-13 21:58:04 [ERROR]: Unsupported language
2024-06-13 21:58:04 [DEBUG]: Cleaning output
2024-06-13 21:58:04 [DEBUG]: jpn_vert
2024-06-13 21:58:04 [ERROR]: Unsupported language
2024-06-13 21:58:05 [DEBUG]: Cleaning output
2024-06-13 21:58:05 [DEBUG]: jpn_vert
2024-06-13 21:58:05 [ERROR]: Unsupported language
2024-06-13 21:58:06 [DEBUG]: Cleaning output
2024-06-13 21:58:06 [DEBUG]: jpn_vert
2024-06-13 21:58:06 [ERROR]: Unsupported language
2024-06-13 21:58:06 [DEBUG]: Cleaning output
2024-06-13 21:58:06 [DEBUG]: jpn_vert
2024-06-13 21:58:06 [ERROR]: Unsupported language
2024-06-13 21:58:07 [DEBUG]: Cleaning output
2024-06-13 21:58:07 [DEBUG]: jpn_vert
2024-06-13 21:58:07 [ERROR]: Unsupported language
2024-06-13 21:58:07 [DEBUG]: Cleaning output
2024-06-13 21:58:07 [DEBUG]: jpn_vert
2024-06-13 21:58:07 [ERROR]: Unsupported language
2024-06-13 21:58:08 [DEBUG]: Cleaning output
2024-06-13 21:58:08 [DEBUG]: jpn_vert
2024-06-13 21:58:08 [ERROR]: Unsupported language
2024-06-13 21:58:08 [DEBUG]: Cleaning output
2024-06-13 21:58:08 [DEBUG]: jpn_vert
2024-06-13 21:58:08 [ERROR]: Unsupported language
2024-06-13 21:58:09 [DEBUG]: Cleaning output
2024-06-13 21:58:09 [DEBUG]: jpn_vert
2024-06-13 21:58:09 [ERROR]: Unsupported language
2024-06-13 21:58:10 [DEBUG]: Cleaning output
2024-06-13 21:58:10 [DEBUG]: jpn_vert
2024-06-13 21:58:10 [ERROR]: Unsupported language
2024-06-13 21:58:11 [DEBUG]: Cleaning output
2024-06-13 21:58:11 [DEBUG]: jpn_vert
2024-06-13 21:58:11 [ERROR]: Unsupported language
2024-06-13 21:58:12 [DEBUG]: Cleaning output
2024-06-13 21:58:12 [DEBUG]: jpn_vert
2024-06-13 21:58:12 [ERROR]: Unsupported language
2024-06-13 21:58:13 [DEBUG]: Cleaning output
2024-06-13 21:58:13 [DEBUG]: jpn_vert
2024-06-13 21:58:13 [ERROR]: Unsupported language
2024-06-13 21:58:14 [DEBUG]: Cleaning output
2024-06-13 21:58:14 [DEBUG]: jpn_vert
2024-06-13 21:58:14 [ERROR]: Unsupported language
2024-06-13 21:58:15 [DEBUG]: Cleaning output
2024-06-13 21:58:15 [DEBUG]: jpn_vert
2024-06-13 21:58:15 [ERROR]: Unsupported language
2024-06-13 21:58:16 [DEBUG]: Cleaning output
2024-06-13 21:58:16 [DEBUG]: jpn_vert
2024-06-13 21:58:16 [ERROR]: Unsupported language
2024-06-13 21:58:17 [DEBUG]: Cleaning output
2024-06-13 21:58:17 [DEBUG]: jpn_vert
2024-06-13 21:58:17 [ERROR]: Unsupported language
2024-06-13 21:58:18 [DEBUG]: Cleaning output
2024-06-13 21:58:18 [DEBUG]: jpn_vert
2024-06-13 21:58:18 [ERROR]: Unsupported language
2024-06-13 21:58:18 [DEBUG]: Cleaning output
2024-06-13 21:58:18 [DEBUG]: jpn_vert
2024-06-13 21:58:18 [ERROR]: Unsupported language
2024-06-13 21:58:19 [DEBUG]: Cleaning output
2024-06-13 21:58:19 [DEBUG]: jpn_vert
2024-06-13 21:58:19 [ERROR]: Unsupported language
2024-06-13 21:58:19 [DEBUG]: Cleaning output
2024-06-13 21:58:19 [DEBUG]: jpn_vert
2024-06-13 21:58:19 [ERROR]: Unsupported language
2024-06-13 21:58:19 [Level 100]: An exception occurred
Traceback (most recent call last):
  File "C:\Users\Skull\Documents\github\Open-Translation\trad.py", line 624, in <module>
    main()
  File "C:\Users\Skull\Documents\github\Open-Translation\trad.py", line 544, in main
    blurbs, height, width = get_blurbs(img, input_language, ocr_mode)
  File "C:\Users\Skull\Documents\github\Open-Translation\trad.py", line 316, in get_blurbs
    text = pytesseract.image_to_string(pil_image, lang=input_language, config=get_params(ocr_mode))
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 423, in image_to_string
    return {
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 426, in <lambda>
    Output.STRING: lambda: run_and_get_output(*args),
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 288, in run_and_get_output
    run_tesseract(**kwargs)
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 262, in run_tesseract
    with timeout_manager(proc, timeout) as error_string:
  File "C:\Python310\lib\contextlib.py", line 135, in __enter__
    return next(self.gen)
  File "C:\Users\Skull\.virtualenvs\Open-Translation-cUeqi7V-\lib\site-packages\pytesseract\pytesseract.py", line 127, in timeout_manager
    yield proc.communicate()[1]
  File "C:\Python310\lib\subprocess.py", line 1149, in communicate
    stdout, stderr = self._communicate(input, endtime, timeout)
  File "C:\Python310\lib\subprocess.py", line 1523, in _communicate
    self.stdout_thread.join(self._remaining_time(endtime))
  File "C:\Python310\lib\threading.py", line 1089, in join
    self._wait_for_tstate_lock()
  File "C:\Python310\lib\threading.py", line 1109, in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
KeyboardInterrupt
